<?xml version="1.0"?>
<root main_tree_to_execute="BehaviorTree">
    <!-- ////////// -->
    <BehaviorTree ID="BehaviorTree">
        <Repeat num_cycles="10">
            <Sequence>
                <Action ID="SleepNode" msec="3000"/>
                <Action ID="Nav2Client" goal="1.5;1.0;0.7;0.7"/>
                <Action ID="SleepNode" msec="3000"/>
                <Action ID="Nav2Client" goal="1.5;-1.0;0.7;0.7"/>
            </Sequence>
        </Repeat>
    </BehaviorTree>
    <!-- ////////// -->
    <TreeNodesModel>
        <Action ID="Nav2Client">
            <input_port name="goal"/>
        </Action>
        <Action ID="SleepNode">
            <input_port name="msec"/>
        </Action>
    </TreeNodesModel>
    <!-- ////////// -->
</root>
